src/CTA.WebForms2Blazor/FileConverters/ConfigFileConverter.cs

```diff
@@ -3,19 +3,57 @@
 using System.Text;
 using System.Threading.Tasks;
 using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.Rules.Actions;
+using CTA.Rules.Models;
+using Newtonsoft.Json.Linq;
+using System.Configuration;
+using System.IO;
+using System.Reflection;
+using Newtonsoft.Json;
+using System.Linq;
 
 namespace CTA.WebForms2Blazor.FileConverters
 {
     public class ConfigFileConverter : FileConverter
     {
+        private const string webConfigFile = "web.config";
+        private string _projectPrefix;
+        private string _fullPath;
+        private string _relativeDirectory;
+
         public ConfigFileConverter(string relativePath) : base(relativePath)
         {
+            var workingDirectory = Environment.CurrentDirectory;
+            _projectPrefix = Directory.GetParent(workingDirectory).Parent.Parent.FullName;
+            _fullPath = Path.Combine(_projectPrefix, RelativePath);
+            _relativeDirectory = Path.GetDirectoryName(RelativePath);
 
         }
 
-        public override Task<IEnumerable<FileInformation>> MigrateFileAsync()
+
+        public override async Task<IEnumerable<FileInformation>> MigrateFileAsync()
         {
-            throw new NotImplementedException();
+            string filename = Path.GetFileName(RelativePath);
+            FileInformation fi;
+
+            if (filename.Equals(webConfigFile))
```

Comment:
You may want to make this comparison case insensitive, I had to do a similar thing when implementing the class converter factory class i.e. `filename.Equals(webConfigFile, StringComparison.InvariantCultureIgnoreCase)`

Reply:
Also, I believe there can be config files other than web.config and package.config so we do we need to add some kind of processing for those too?